### ISSUE TITLE
Better opportunity colour scale

### ISSUE THREAD

**Issue by abulte** [abulte/velo-weather#6]
Re #3 

The opportunity indicator seemed more readable to me before the gradient change (everything else looks better!). The "dirty" green conveyed some meaning. WDYT @joachimesque?

## Before

<img width="1228" alt="Capture d’écran 2021-11-02 à 17 28 25" src="https://user-images.githubusercontent.com/119625/139906269-579a2a87-b963-4d7d-bfaa-de910639be2e.png">

## After

<img width="1232" alt="Capture d’écran 2021-11-02 à 17 27 51" src="https://user-images.githubusercontent.com/119625/139906282-38b151e0-3ee0-4cec-97d8-fdd404f492c7.png">

**Comment by abulte** [comment]
Somehow better with https://github.com/abulte/velo-weather/commit/2955b5614a9f7664362acdf937cc2b0f9c9b2ee4

**Comment by joachimesque** [comment]
You're right! The saturation is at 100% all across the gradient, so the light green is very flashy… I'll have to see if there's a way to add values inside the gradient
